doc/sphinx-guides/source/installation/config.rst

```diff
@@ -991,123 +897,83 @@ The chart below shows examples from each possible combination of parameters from
 
-Otherwise, if ``:DataFilePIDFormat`` is set to *INDEPENDENT*, then each file will be assigned a PID with the next number in the overall sequence, regardless of what dataset it is in. If the file is created after a dataset with the PID 100001, then the file will be assigned the PID 100002. This option is functional, but it is not a recommended use case.
+Otherwise, if ``DataFilePIDFormat`` is set to *INDEPENDENT*, then each file will be assigned a PID with the next number in the overall sequence, regardless of what dataset it is in. If the file is created after a dataset with the PID 100001, then the file will be assigned the PID 100002. This option is functional, but it is not a recommended use case.
 
-Note that in either case, when using the ``sequentialNumber`` option, datasets and files share the same database sequence that was created as part of the setup described in ``:IdentifierGenerationStyle`` above.
+Note that in either case, when using the ``sequentialNumber`` option, datasets and files share the same database sequence that was created as part of the setup described in ``IdentifierGenerationStyle`` above.
 
-.. _:FilePIDsEnabled:
+.. _FilePIDsEnabled:
 
-:FilePIDsEnabled
+FilePIDsEnabled
 ++++++++++++++++
 
-Toggles publishing of file-based PIDs for the entire installation. By default this setting is absent and Dataverse assumes it to be true.
+Toggles publishing of file-based PIDs for the entire installation. By default this setting assumes it to be true.
 
-If you don't want to register file-based PIDs for your installation, set:
-
-``curl -X PUT -d 'false' http://localhost:8080/api/admin/settings/:FilePIDsEnabled``
+If you don't want to register file-based PIDs for your installation, set it to false.
 
 Note: File-level PID registration was added in 4.9 and is required until version 4.9.3.
 
-:IndependentHandleService
-+++++++++++++++++++++++++
+.. _IndependentHandleService:
 
-Specific for Handle PIDs. Set this setting to true if you want to use a Handle service which is setup to work 'independently' (No communication with the Global Handle Registry). 
-By default this setting is absent and Dataverse assumes it to be false.
+IndependentHandleService
++++++++++++++++++++++++++++
 
-``curl -X PUT -d 'true' http://localhost:8080/api/admin/settings/:IndependentHandleService``
+Specific for Handle PIDs. Set this setting to true if you want to use a Handle service which is setup to work 'independently' (No communication with the Global Handle Registry). 
+By default this setting is false.
 
-:ApplicationTermsOfUse
+ApplicationTermsOfUse
 ++++++++++++++++++++++
 
 Upload an HTML file containing the Terms of Use to be displayed at sign up. Supported HTML tags are listed under the :doc:`/user/dataset-management` section of the User Guide.
 
-``curl -X PUT -d@/tmp/apptou.html http://localhost:8080/api/admin/settings/:ApplicationTermsOfUse``
+``ApplicationTermsOfUse=@/tmp/apptou.html``
```

Comment:
You can't provide path to a file in properties file, you instead provide raw text.